app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -99,21 +99,6 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         )
     }
 
-    private val browsingModeListener = object : BrowsingModeListener {
-        override fun onBrowsingModeChange(newMode: BrowsingMode) {
-            themeManager.currentTheme = newMode
-        }
-    }
-
-    override fun applyOverrideConfiguration(overrideConfiguration: Configuration?) {
```

Comment:
This is reverted from: https://github.com/mozilla-mobile/fenix/commit/1b7cb117cade9500e0130d7f3177b115ae7bb626#diff-44fc84568dc7e79b2cb8921c2135183a